packages/components/src/modifiers/hds-clipboard.ts

```diff
@@ -6,8 +6,8 @@
 import { modifier } from 'ember-modifier';
 import { assert, warn } from '@ember/debug';
 
-export const getTextToCopy = (text) => {
-  let textToCopy;
+export const getTextToCopy = (text: { toString: () => string }) => {
```

Comment:
Considered the logic below, in this function, I _think_ we should do something different here. 
@alex-ju what do you think?